package.json

```diff
@@ -58,27 +59,23 @@
     "node": ">=0.10"
   },
   "dependencies": {
-    "assert-plus": "^0.1.5",
-    "backoff": "^2.4.0",
+    "assert-plus": "0.1.5",
```

Comment:
What was the reason behind saving the exact dependency? As a library, we tend to want to be flexible for others to pull in any version that we support.